examples/python/pincell_depletion/run_depletion.py

```diff
@@ -0,0 +1,161 @@
+import openmc
+import numpy as np
+
+###############################################################################
+#                      Simulation Input File Parameters
+###############################################################################
+
+# OpenMC simulation parameters
+batches = 100
+inactive = 10
+particles = 1000
+
+# Depletion simulation parameters
+time_step = 1.*24*60*60 # s
+final_time = 15.*24*60*60 # s
+time_steps = np.full(np.int(final_time / time_step), time_step)
```

Comment:
`np.full(final_time // time_step, time_step)` would have the same effect

Reply:
If final_time and time_step are integers it will, else final_time // time_step will be a float and it won't work. Changed to your version.